### ISSUE TITLE
[BUG] Add support for defining pointer to pointers. 

### ISSUE THREAD

**Issue by leejy12** [hsutter/cppfront#78]
Hi, I am very interested in this project's future!

Cppfront fails to compile this code
```cpp
main: () -> int = {
    a:     int = 2;
    pa:   *int = a&;
    ppa: **int = pa&;
    return a*pa**ppa**; // 8
}
```

As of now, `ppa: **int = pa&;` does not work.

**Comment by filipsajdak** [comment]
That is true. As a workaround you can use type aliases.

**Comment by hsutter** [comment]
@leejy12 Thanks, fixed with commit e1b57cf026041cef33d4f95ae230513ada29fa51 .

This now works (note spacing around `*`, otherwise it's your example):

```
main: () -> int = {
    a:     int = 2;
    pa:   *int = a&;
    ppa: **int = pa&;
    std::cout << a * pa* * ppa**; // 8
}
```

Thanks!
